### PR TITLE
[Cleanup] Use DataBinding for TwoLineToolbar

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Replace toolbar title/subtitle setter to layout using DataBinding

Edit:
Also, fixes center alignment of title

### CODE REVIEW

**Comment by shobhitagarwal1612** [comment]
Already made that fix 1384266

**Comment by gino-m** [comment]
Awesome, thank you! @dturner could you PTAL for approval?

File: gnd/src/main/java/com/google/android/gnd/ui/common/TwoLineToolbar.java [hunk 1]
```diff
@@ -41,12 +41,16 @@ public TwoLineToolbar(Context context, @Nullable AttributeSet attrs) {
...
-  public void setTitle(String title) {
+  public void setTitle(@Nullable String title) {
     titleText.setText(title);
   }
 
-  public void setSubtitle(String subtitle) {
+  public void setSubtitle(@Nullable String subtitle) {
     subtitleText.setText(subtitle);
-    subtitleText.setVisibility(subtitle.isEmpty() ? View.GONE : View.VISIBLE);
+    if (subtitle == null) {
```
Comment (dturner) [inline gnd/src/main/java/com/google/android/gnd/ui/common/TwoLineToolbar.java]: To avoid the ternary operator below you could change this to `if(subtitle == null || subtitle.isEmpty()){ // GONE } else { // VISIBLE }`.

File: gnd/src/main/java/com/google/android/gnd/MainActivity.java [hunk 2]
```diff
@@ -45,6 +44,7 @@
 import dagger.android.support.DaggerAppCompatActivity;
 import javax.inject.Inject;
 import javax.inject.Singleton;
+import timber.log.Timber;
```
Comment (dturner) [inline gnd/src/main/java/com/google/android/gnd/MainActivity.java]: nit: These logging changes would be better in a different PR since they are unrelated to the DataBinding change.

**Comment by shobhitagarwal1612** [comment]
@dturner Fixed. PTAL!

File: gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java [hunk 3]
```diff
@@ -362,7 +359,7 @@ private int getSelectedProjectIndex(Project activeProject) {
 
   private void onShowAddFeatureDialogRequest(Point location) {
     if (!Loadable.getValue(viewModel.getActiveProject()).isPresent()) {
-      Log.e(TAG, "Attempting to add feature while no project loaded");
+      Timber.e("Attempting to add feature while no project loaded");
```
Comment (dturner) [inline gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java]: Still seeing a few of these Timber changes but it's super picky so LGTM.

Comment (shobhitagarwal1612) [reply gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java]: I just thought of cleaning it up while we were editing the file. But will remember the suggestion for future PRs. Thanks

Comment (gino-m) [reply gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java]: +1 to cleaning these up in bulk in a separate PR, but ok for this one.
